packages/react/src/components/MenuButton/SingleMenuButton.jsx

```diff
@@ -31,7 +31,7 @@ const propTypes = {
    */
   iconDescription: PropTypes.string.isRequired,
 
-  testID: PropTypes.string.isRequired,
+  testId: PropTypes.string.isRequired,
```

Comment:
No deprecation of testID?

Reply:
It's an internal component that's not exported, so I just changed it and updated the references.